packages/calling-component-bindings/package.json

```diff
@@ -29,19 +29,19 @@
   },
   "dependencies": {
     "@azure/communication-common": "1.1.0",
-    "@internal/acs-ui-common": "1.0.1-beta.2",
-    "@internal/calling-stateful-client": "1.0.1-beta.2",
-    "@internal/react-components": "1.0.1-beta.2",
+    "@internal/acs-ui-common": "1.1.0",
+    "@internal/calling-stateful-client": "1.1.0",
+    "@internal/react-components": "1.1.0",
     "memoize-one": "~5.2.1",
     "reselect": "~4.0.0"
   },
   "peerDependencies": {
-    "@azure/communication-calling": "1.3.2-beta.1 || 1.3.2",
+    "@azure/communication-calling": "1.3.2",
```

Comment:
Note: These lines(choose only one dependencies) need to be reverted when we merge this back in

Reply:
So the stable deps need to be chosen .